release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -80,15 +80,18 @@ For more information, see xref:../backup_and_restore/hibernating-cluster.adoc#hi
 [id="ocp-release-notes-extensions_{context}"]
 === Extensions ({olmv1})
 
-Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped grow and enable a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces {olmv1}, the next-generation Operator Lifecycle Manager, as a General Availability feature, designed to improve how you manage Operators on {product-title}.
+[id="ocp-release-notes-extensions-olmv1-ga_{context}"]
+==== {{olmv1-first} (General Availability)
+
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release and has helped grow and enable a substantial ecosystem of solutions and advanced workloads running as Operators. {product-title} {product-version} introduces {olmv1}, the next-generation Operator Lifecycle Manager, as a General Availability (GA) feature, designed to improve how you manage Operators on {product-title}.
 
 Previously available as a Technology Preview feature only, this updated framework evolves many of the concepts that have been part of previous versions of {olmv0} by simplifying Operator management, enhancing secrutiy, and boosting reliability.
 
 [NOTE]
 ====
-The documentation for {olmv1} exists as a stand-alone guide called xref:../extensions/index.adoc#olmv1-about[Extensions]. Previously, {olmv1} documentation was a subsection of the xref:../operators/index.adoc#operators-overview[Operators] guide, which otherwise documents the {olmv0} feature set.
+Starting in {product-title} {product-version}, {olmv1} is now enabled by default, alongside the {olmv0}. Cluster administrators can toggle installation of {olmv1} by using cluster capabilities at installation time.
 
-The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
+The {olmv0} remains fully supported throughout the {product-title} 4 lifecycle.
 ====
```

Comment:
This is only weird like this because I added a new NOTE here and moved the existing NOTE to the end of this subsection below.

Reply:
Are you saying we are repeating the information here?

Reply:
No, sorry, just explaining that the diff is strange because of swapping two unrelated `[NOTE]` boxes.